.settings/org.eclipse.buildship.core.prefs

```diff
@@ -0,0 +1,13 @@
+arguments=
+auto.sync=false
+build.scans.enabled=false
+connection.gradle.distribution=GRADLE_DISTRIBUTION(WRAPPER)
+connection.project.dir=
+eclipse.preferences.version=1
+gradle.user.home=
+java.home=C\:/Program Files/Amazon Corretto/jdk11.0.8_10
+jvm.arguments=
+offline.mode=false
+override.workspace.settings=true
+show.console.view=true
+show.executions.view=true
```

Comment:
revert these changes

Reply:
Ok, sir, I didn't notice that. Thanks for telling me.